.github/actions/.gitignore

```diff
@@ -0,0 +1,3 @@
+# ignore dependency packages
```

Comment:
speaking of ignore files, you should update `.vscodeignore` to ignore the `.github` folder so it doesn't end up in our VSIX.

Reply:
This is not valid, correct?  `.github. is at the root of the repo.  The extension is built from the Extension subdirectory, which is also where the .vscodeignore file is.

Reply:
You're right. I've been working in 2 other repos where .github/ and src/ are both in the root.  I forgot about how this repo doesn't do that.